themes/cardinal_service/lib/components/opportunity-row/opportunity_row.scss

```diff
@@ -1 +1,71 @@
 @charset 'UTF-8';
+
+@import '../../scss/config';
+
+.pattern-preview__markup {
```

Comment:
none of the styles are usable because of this class. you'll want to only target classes that you have defined within the twig file. don't rely on anything else being there

Reply:
Thanks @pookmish :) I updated the twig template — please let me know if further adjustments are needed!